### PR TITLE
Givers stats

### PR DESCRIPTION
**Pull request by mateuszkocz**

### CODE REVIEW

**Comment by ulaulala** [comment]
Please use WIP label instead of "[WIP]" in title :)

File: src/components/DateNavigation.js [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+    createUrl (month, year) {
+        return `${process.env.PUBLIC_URL}/stats/givers/${year}/${month}`
+    }
+    createPrevLink (currentMonth, currentYear) {
+        const month = currentMonth === 1 ? 12 : currentMonth - 1
+        const year = currentMonth === 1 ? currentYear - 1 : currentYear
+        return year < getBaseYear() ? '' : this.createUrl(month, year)
+    }
+    createNextLink (currentMonth, currentYear) {
+        const month = currentMonth === 12 ? 1 : currentMonth + 1
```
Comment (kgajowy) [inline src/components/DateNavigation.js]: What do you think about something more fancy? I know those below are less readable when you see them for the very first time , so you may consider those below as an interesting alternative
next:
image
prev:
image

File: src/utils/months.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
+import {getCurrentYear} from "./years";
+const currentMonth = (new Date()).getMonth()
+const months = [
```
Comment (kgajowy) [inline src/utils/months.js]: I may be missing something but why is this necessary?


image

If you really need this mapping written by hand, it may be easier to use the built-in functions above, which will include locale as well.

**Comment by ulaulala** [comment]
Hey, desktop looks really cool :D but it's totally not responsive. I wonder it could be better to use select tags for years and months - what do you think about it?

File: src/utils/months.js (hunk 2 above)
Comment (mateuszkocz) [reply src/utils/months.js]: Googling "months and abbreviation json" was faster than writing code. 🤷‍♂️ 

This whole functionality will most likely be removed when API will start sending months as numbers (@dotadota counting on you!).

**Comment by mateuszkocz** [comment]
I can't promise anything regarding the responsiveness as I don't know how long I'll be able to work on this app. Will try to cook something up. Most likely those selects.

File: src/utils/months.js (hunk 2 above)
Comment (mateuszkocz) [reply src/utils/months.js]: Changed some stuff, so now your idea is used in the code. 😘

**Comment by mateuszkocz** [comment]
@kgajowy another review plz
@ulaulala review plz
